### PR TITLE
login SP graph permission required

### PR DESCRIPTION
**Pull request by afengli**
if login sp does not have access to graph permission, the command will result in error like:

Operation failed with status: 'Forbidden'. Details: 403 Client Error: Forbidden for url: https://graph.windows.net/xxxxxxxx-0000-0000-0000-000000000000/getObjectsByObjectIds?api-version=1.6

the login SP need to be configured to give permission on Microsoft Graph .

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [ ] Each command and parameter has a meaningful description.
- [ ] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

File: src/command_modules/azure-cli-role/azure/cli/command_modules/role/_help.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
 helps['ad sp create-for-rbac'] = """
     type: command
     short-summary: Create a service principal and configure its access to Azure resources.
+    Note: if cli login is using service principal, the service principal should have read and write access to Microsoft Graph appliation in the AAD tenant.
```
Comment (yugangw-msft) [inline src/command_modules/azure-cli-role/azure/cli/command_modules/role/_help.py]: Could you cross check whether this is all needed? Based on my experience, 2 things need to happen
1.  The permission should be granted to `Windows Azure Active Directory`, rather `Microsoft Graph`. These are 2 different resources
2. Consent the permission required by the service principal. If you are in portal, you can just click the "grant permission" button

If this is what you went through to get things work as well, could you update the text here?

**Comment by afengli** [comment]
Thank you @yugangw-msft. Yes that is correct, two application permission needs to be configured. I have  updated the pull request, please check.

**Comment by yugangw-msft** [comment]
@afengli, I don't think  `Microsoft Graph` is needed; also unless the service principal needs to run commands under `az ad`, it doesn't really need to access any other resources

**Comment by yugangw-msft** [comment]
Please reopen when the change is ready

**Pull request closed without merging**